samples/bot-conversation/nodejs/bots/teamsConversationBot.js

```diff
@@ -70,6 +70,21 @@ class TeamsConversationBot extends TeamsActivityHandler {
                 // Save information about the sent message and its ID (resourceResponse.id).
             }));
         });
+
+        this.onTeamsMessageEditEvent(async (context, next) => {
+            await context.sendActivity("Message is edited");
+            next();
+          });
```

Comment:
await context.sendActivity("Message is edited.");

await context.sendActivity("Message is undeleted.");

await context.sendActivity("Message is deleted.");

Reply:
Bot message or ?

Reply:
Its a generic message as provided by code snippets from doc team